UIKitDZ/UIKitDZ/PizzaSelectionViewController.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  PizzaSelectionViewController.swift
+//  UIKitDZ
+//
+//  Created by Maxim on 29.09.2022.
+//
+
+import UIKit
+///  Контроллер выбора пиццы
+final class PizzaSelectViewController: UIViewController {
+
+    let pizzaNameOneLabel = UILabel()
```

Comment:
```suggestion
   private let pizzaNameOneLabel = UILabel()
```
private - пиши везде ниже